src/basic/btrfs-util.c

```diff
@@ -418,12 +418,7 @@ static int btrfs_ioctl_search_args_compare(const struct btrfs_ioctl_search_args
         if (args->key.min_type > args->key.max_type)
                 return 1;
```

Comment:
Here, we can also use `CMP()`...

Reply:
yupp, but the cociccenelle check is harder, so i was lazy for for now... but yes, we should definitely add that later, too